homeassistant/components/rituals_perfume_genie/binary_sensor.py

```diff
@@ -15,6 +21,34 @@
 from .entity import DiffuserEntity
 
 
+@dataclass
+class RitualsentityDescriptionMixin:
+    """Mixin values for Rituals entities."""
+
+    is_on_fn: Callable[[Diffuser], bool]
+
+
+@dataclass
+class RitualsBinarySensorEntityDescription(
+    BinarySensorEntityDescription, RitualsentityDescriptionMixin
+):
+    """Class describing Rituals binary sensor entities."""
+
+    has_fn: Callable[[Diffuser], bool] = lambda _: True
```

Comment:
I'm surprised that you are setting a default here, since it is overriden in all instances.

Reply:
True, pushed a change.